### DISCUSSION TITLE
Who was working on what, how do we configure launch files for it?

### DISCUSSION THREAD

**Discussion by AnthonyZhOon** [Monash-Connected-Autonomous-Vehicle/autoware#25] [General]
@Monash-Connected-Autonomous-Vehicle/avp-team, @Robin-Chea is working on setting up launch files for everything to make it easy to run our code consistently. So we're reaching out to find who was working on CARLA and AWSIM, as well as the people who wrote the nodes for running the Twizy and ENV200.

**Comment by dylan-gonzalez** [comment]
@npnquang has been the main one involved with AWSIM
Currently, @Jokua (William Tioe) is working on [CARLA](https://github.com/orgs/Monash-Connected-Autonomous-Vehicle/projects/8/views/15?pane=issue&itemId=44231187)

What do you mean by the nodes for the Twizy and ENV200? Do you mean the sd_vehicle_interface package? That's made by Streetdrone, and currently @dtonda8 @jin-ramen @jaimasters are working on making it run better.
